grails-app/domain/org/pih/warehouse/shipping/ShipmentItem.groovy

```diff
@@ -256,6 +256,10 @@ class ShipmentItem implements Comparable, Serializable {
         return inventoryItem?.lotStatus == LotStatusCode.RECALLED
     }
 
+    Integer getQuantityInvoiced() {
```

Comment:
not a change request, but just FYI - this doesn't need to be a "heavier" `Integer`, but could be `int`, since this cannot be `null`. (You made a fallback for `0`.